internal/resources-validation.yml

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) 2019 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+parameters:
+  # Git ref of tk-toolchain to use.
+  tk_toolchain_ref: master
+  # Git ref of tk-core to use.
+  tk_core_ref: master
+
+jobs:
+# Spin up a Linux VM for validating code.
+- job: png_resources_validation
+  displayName: PNG Resources Validation
+  pool:
+    vmImage: 'ubuntu-latest'
+
+  steps:
+    - script: |
+        sudo apt update
+        sudo apt install -y pngcrush
+      displayName: 'Install Dependencies'
+    - script: |
+        # Skip PNG profile validation on non-ubuntu-latest agents
+        if [[ "$(Agent.OS)" != "Linux" ]]; then
+          echo "Skipping PNG profile validation as it is not running on ubuntu-latest agent"
+          exit 0
+        fi
+        # Navigate to the directory containing the PNG files
+        cd $(Build.SourcesDirectory)/resources
```

Comment:
Probably we can use `workingDirectory` instead: https://learn.microsoft.com/en-us/azure/devops/pipelines/tasks/reference/cmd-line-v2?view=azure-pipelines&tabs=yaml#inputs